android/src/test/java/com/amplitude/android/IdentifyInterceptorTest.kt

```diff
@@ -0,0 +1,286 @@
+package com.amplitude.android
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import com.amplitude.android.plugins.AndroidLifecyclePlugin
+import com.amplitude.common.android.AndroidContextProvider
+import com.amplitude.core.Constants
+import com.amplitude.core.events.BaseEvent
+import com.amplitude.core.events.EventOptions
+import com.amplitude.core.events.Identify
+import com.amplitude.core.events.IdentifyOperation
+import com.amplitude.core.utilities.toEvents
+import io.mockk.every
+import io.mockk.mockkConstructor
+import io.mockk.mockkStatic
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.launch
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import okhttp3.mockwebserver.RecordedRequest
+import org.json.JSONObject
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.lang.Thread.sleep
+import java.util.concurrent.TimeUnit
+
+@RunWith(RobolectricTestRunner::class)
+class IdentifyInterceptorTest {
+    private lateinit var server: MockWebServer
+    private lateinit var amplitude: Amplitude
+
+    @ExperimentalCoroutinesApi
+    @Before
+    fun setup() {
+        server = MockWebServer()
+        server.start()
+        val context = ApplicationProvider.getApplicationContext<Context>()
+        mockContextProvider()
+
+        val apiKey = "1cc2c1978ebab0f6451112a8f5df4f4e"
```

Comment:
Is this a real API key? If so we should probably replace with a placeholder, mock.

Reply:
I guess this is fake, will change to a placeholder